src/arb/arb.jl

```diff
@@ -569,7 +569,7 @@ end
 @doc Markdown.doc"""
     ball(x::arb, y::arb)
 
-Constructs an `arb` via $x_m \pm (|x_r| + |y_m| + |y_r|)$, given the pair
+Constructs an Arb ball $x_m \pm (|x_r| + |y_m| + |y_r|)$, given the pair
```

Comment:
```suggestion
Constructs an arb ball enclosing $x_m \pm (|x_r| + |y_m| + |y_r|)$, given the pair
```
I think the enclosing part is necessary.

Reply:
Sounds good! I changed "arb" to "Arb", since my understanding is that it is either written "`arb`" or "Arb".

Reply:
Sure, feel free to change it to Arb or `arb`.